src/main/webapp/resources/js/pages/projects/create/CreateProjectSamples.js

```diff
@@ -0,0 +1,126 @@
+import {
+  Alert,
+  Button,
+  Checkbox,
+  Empty,
+  Form,
+  Input,
+  Space,
+  Table,
+  Typography,
+} from "antd";
+import React from "react";
+import { useGetCartSamplesQuery } from "../../../apis/cart/cart";
+import { IconExperiment } from "../../../components/icons/Icons";
+import { SampleDetailViewer } from "../../../components/samples/SampleDetailViewer";
+import { blue6 } from "../../../styles/colors";
+
+/**
+ * Component to render samples that are in the cart (if any).
+ * User can select which samples to add to the new project.
+ * @param {Object} form - Ant Design form API
+ * @returns {JSX.Element}
+ * @constructor
+ */
+export function CreateProjectSamples({ form }) {
+  const { data: samples = {}, isLoading } = useGetCartSamplesQuery();
+  const [organismFilter, setOrganismFilter] = React.useState([]);
+  const [selected, setSelected] = React.useState([]);
+  const [lock, setLock] = React.useState(false);
+
+  React.useEffect(() => {
+    const exists = {};
+    samples.unlocked?.forEach((sample) => {
```

Comment:
Add comment as to what is happening here

Reply:
Updated in 0cbb435b5b4e7a1bcd010f5014b7cd0d31a7e6ef